### PR TITLE
Prevent `update_current_user_after_login` from being called 3 times in Dashboard

### PR DESCRIPTION
**Pull request by josemigallas**
This is called 3 times in the dashboard:
```
  SQL (0.8ms)  UPDATE `users` SET `users`.`remember_token_expires_at` = NULL, `users`.`remember_token` = NULL WHERE `users`.`id` = 2
   (0.2ms)  BEGIN
  SQL (0.4ms)  UPDATE `user_sessions` SET `accessed_at` = '2022-03-28 11:59:18', `updated_at` = '2022-03-28 11:59:18' WHERE `user_sessions`.`id` = 7
   (0.4ms)  COMMIT
```

### CODE REVIEW

File: app/controllers/provider/admin/dashboard/widget_controller.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
+# frozen_string_literal: true
+
 class Provider::Admin::Dashboard::WidgetController < Provider::Admin::BaseController
   helper_method :widget, :current_range, :previous_range
   before_action :load_widget
 
+  skip_after_action :update_current_user_after_login
```
Comment (josemigallas) [inline app/controllers/provider/admin/dashboard/widget_controller.rb:7]:
```suggestion
  skip_after_action :update_current_user_after_login # already called by main controller
```

File: test/integration/provider/admin/dashboards_controller_test.rb [hunk 2]
```diff
@@ -29,4 +29,12 @@ class Provider::Admin::DashboardsControllerTest < ActionDispatch::IntegrationTes
     get provider_admin_dashboard_path
     assert_xpath(xpath_selector, element_text)
   end
+
+  test '#update_current_user_after_login called once' do
+    Admin::Api::BaseController.expects(:update_current_user_after_login).once
```
Comment (akostadinov) [inline test/integration/provider/admin/dashboards_controller_test.rb:34]: Probably this:
```suggestion
    Admin::Api::BaseController.any_instance.expects(:update_current_user_after_login).once
```

File: app/lib/authenticated_system.rb [hunk 3]
```diff
@@ -30,7 +30,6 @@ def clear_current_user
     authenticated_request.reset!
   end
 
-  # Feel freee to add skip_after_action for this method in controllers for stats or api transactions
```
Comment (akostadinov) [inline app/lib/authenticated_system.rb:33]: why remove this line?

Comment (josemigallas) [reply app/lib/authenticated_system.rb:33]: Yeah.. IDK, I thought this PR solved this comment but I guess it's still valid.

File: test/integration/provider/admin/dashboards_controller_test.rb (hunk 2 above)
Comment (josemigallas) [reply test/integration/provider/admin/dashboards_controller_test.rb]: I can't make it fail! Without skipping the method, it is called 3 times according to the server logs
- Provider::Admin::DashboardsController
- Provider::Admin::Dashboard::PotentialUpgradesController
- Provider::Admin::Dashboard::NewAccountsController

However, the test only satisfies the following when added in setup, before login:
```ruby
Provider::Admin::DashboardsController.any_instance.expects(:update_current_user_after_login).times(2)
```

And this is always true, with or without skipping:
```ruby
Provider::Admin::Dashboard::PotentialUpgradesController.any_instance.expects(:update_current_user_after_login).once
Provider::Admin::Dashboard::NewAccountsController.any_instance.expects(:update_current_user_after_login).once
```

**Comment by secretsurf** [comment]
Josemi, i'm re-opening this. And let's reconsider this bot...

**Pull request closed without merging**